.circleci/config.yml

```diff
@@ -53,7 +53,7 @@ workflows:
 jobs:
   build-ubuntu-image:
     environment:
-      IMAGE_NAME: haugene/transmission-openvpn
+      IMAGE_NAME: colinwebber/transmission-openvpn
```

Comment:
this file should not be part of the commit as this will link to the wrong image

Reply:
Ah, so , if you remove the commit b97d817 then it will be fine

Reply:
Apologies, I did that for testing purposes but committed to the same branch. I'm going to create a new pull request to avoid any remnants.